### PR TITLE
[Feature] OSF for meetings upgrades [OSF-4693]

### PR DESCRIPTION
**Pull request by Ghalko**
## Purpose

Replaces #4505 

Adds new columns to the display of meetings within the table in order to facilitate better meeting representation.

osf.io/meetings
screen shot 2016-05-24 at 1 42 09 pm

admin/meetings/
screen shot 2016-05-24 at 1 41 27 pm

## Changes

- Add location of conference
- Add start and end dates of conference
- Display the preceding in two appropriate columns
- Remove the active column from display, but not from db
- Leave sorting alone unlike original specs (approved by Product)

Additionally
- Add columns to admin forms for faster updating.

## Side effects
People will be able to see where conferences are without having to click through!

## Ticket

https://openscience.atlassian.net/browse/OSF-4693

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Okay, I've closed the other PR and will review this one instead. I was confused about how you were dealing with the admin stuff.

**Comment by brianjgeiger** [comment]
Please properly fill out PR description.

File: scripts/populate_conferences.py [hunk 1]
```diff
@@ -92,11 +111,14 @@ def main():
...
         ),
+        'location': 'Berkeley, CA',
+        'start_date': 'Dec 11 2014',
+        'end_date': 'Dec 12 2014',
         'active': False,
         'admins': [
             'gkroll@berkeley.edu',
             'awais@berkeley.edu',
-        ],
+         ],
```
Comment (brianjgeiger) [inline scripts/populate_conferences.py]: Obvs minor, but are you sure this space is correct here?

File: scripts/populate_conferences.py [hunk 2]
```diff
@@ -744,18 +890,28 @@ def main():
 }
 
 
-def populate_conferences():
+def populate_conferences(dev=False):
```
Comment (brianjgeiger) [inline scripts/populate_conferences.py]: 👍

File: website/static/js/meetings.js [hunk 3]
```diff
@@ -46,13 +52,35 @@ function Meetings(data) {
...
+                    }
                 },
                 {
-                    data: 'active', // Data field name
-                    sortInclude: true,
-                    filter : false,
-                    custom: function() { return item.data.active ? 'Yes' : 'No'; }
+                    data: 'location',
+                    sortInclude : true,
+                    custom : function() {
```
Comment (brianjgeiger) [inline website/static/js/meetings.js]: Can this be item.data.location instead of an anonymous function?

**Comment by brianjgeiger** [comment]
Okay, I'll know better once the PR description is descriptive, but should just be a minor change to a field in an object and maybe a space.

**Comment by brianjgeiger** [comment]
Pass done. 🐧

File: scripts/populate_conferences.py (hunk 1 above)
Comment (Ghalko) [inline scripts/populate_conferences.py]: I don't think it was intended, though I did come across other spacing issues. This was also reviewed by Chad before I got it.

Space removed.

File: website/static/js/meetings.js (hunk 3 above)
Comment (Ghalko) [inline website/static/js/meetings.js]: Custom, it seems, is expecting a function while it is building the table.

Comment (brianjgeiger) [inline website/static/js/meetings.js]: Weird, but okay.

**Comment by brianjgeiger** [comment]
Looks good. Pass done. 🎉 🐧 🎉

**Pull request merged**